### DISCUSSION TITLE
Sidra kyc

### DISCUSSION THREAD

**Discussion by Shazadriaz9595** [blockscout/blockscout#10472] [General]
Sidra KYC is not being done which is facing many difficulties. What is its condition?

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.

**Comment by Bilalalam22** [comment]
Respected Sidra Chain Team,

Kindly I am Waiting for My KYC Verification Since  2 Months So Please Take some Action To Verify My KYC In sidra Bank

Thanks For Your Support

My Username
@bilalalam2

And My Email ID
bilalalam2@gmail.com

Please Verify My KYC As soon as Possible Thankyou.

Regards.

Name Bilal Alam

**Comment by mdalimul** [comment]
Respected Sidra Chain Team,

Kindly I am Waiting for My KYC Verification Since 2 Months So Please Take some Action To Verify My KYC In sidra Bank

Thanks For Your Support

My Username
@akhi647

And My Email ID
[arshiyabasherakhi31124@gmail.com]

Please Verify My KYC As soon as Possible Thankyou.

Regards.

Name Akhi Akter
